src/bindings/python/wheel/setup.py

```diff
@@ -119,6 +119,13 @@
         "rpath": LIBS_RPATH,
         "binary_dir": OPENVINO_BUILD_DIR,
     },
+    "pytorch_libs": {
+        "name": "pytorch",
+        "prefix": "libs.pytorch",
+        "install_dir": OV_RUNTIME_LIBS_DIR,
+        "rpath": LIBS_RPATH,
+        "binary_dir": OPENVINO_BUILD_DIR,
+    },
```

Comment:
@ilya-lavrenov, would we like to include the frontend into wheel/release now?

Reply:
Wheel is needed